tests/validations/test_where_filters_are_parseable.py

```diff
@@ -0,0 +1,309 @@
+import copy
+import logging
+
+import pytest
+
+from dbt_semantic_interfaces.implementations.filters.where_filter import (
+    PydanticWhereFilter,
+    PydanticWhereFilterIntersection,
+)
+from dbt_semantic_interfaces.implementations.saved_query import (
+    PydanticSavedQuery,
+    PydanticSavedQueryQueryParams,
+)
+from dbt_semantic_interfaces.implementations.semantic_manifest import (
+    PydanticSemanticManifest,
+)
+from dbt_semantic_interfaces.test_utils import (
+    check_no_errors_or_warnings,
+    check_only_one_error_with_message,
+    check_only_one_warning_with_message,
+    find_metric_with,
+)
+from dbt_semantic_interfaces.validations.semantic_manifest_validator import (
+    SemanticManifestValidator,
+)
+from dbt_semantic_interfaces.validations.validator_helpers import (
+    SemanticManifestValidationException,
+)
+from dbt_semantic_interfaces.validations.where_filters import (
+    WhereFiltersAreParseableRule,
+)
+
+logger = logging.getLogger(__name__)
+
+
+# ------------------------------------------------------------------------------
```

Comment:
this section is moved from existing tests.  I renamed a few, and the calls to `deepcopy` became `copy.deepcopy`, but they are not functionally changed.

Reply:
Ok sweet - this addresses my last comment!